posthog/session_recordings/sql/session_replay_event_sql.py

```diff
@@ -53,7 +55,11 @@
     distinct_id VARCHAR,
     min_first_timestamp SimpleAggregateFunction(min, DateTime64(6, 'UTC')),
     max_last_timestamp SimpleAggregateFunction(max, DateTime64(6, 'UTC')),
+    -- store the first url of the session so we can quickly show that in playlists
     first_url AggregateFunction(argMin, Nullable(VARCHAR), DateTime64(6, 'UTC')),
+    -- but also store each url so we can query by visited page without having to scan all events
+    -- despite the name we can put mobile screens in here as well to give same functionality across platforms
+    all_urls SimpleAggregateFunction(groupUniqArrayArray, Array(String)),
```

Comment:
from a clickhouse review perspective - currently we have to query the events table to see if someone visited a particular URL but it's a very common query, let's keep the URLs on this table so we can avoid the subquery as much as possible